pkg/assets/assets.go

```diff
@@ -67,13 +85,37 @@ func MustCreateAssetFromTemplate(name string, template []byte, config interface{
 }
 
 func assetFromTemplate(name string, tb []byte, data interface{}) (Asset, error) {
-	tmpl, err := template.New(name).Parse(string(tb))
+	bs, err := renderFile(name, tb, data)
 	if err != nil {
 		return Asset{}, err
 	}
-	var buf bytes.Buffer
-	if err := tmpl.Execute(&buf, data); err != nil {
-		return Asset{}, err
+	return Asset{Name: name, Data: bs}, nil
+}
+
+// LoadFilesRecursively returns a map from relative path names to file content.
+func LoadFilesRecursively(dir string) (map[string][]byte, error) {
+	files := map[string][]byte{}
+	err := filepath.Walk(dir,
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			if info.IsDir() {
+				return nil
+			}
+
```

Comment:
do we want to load only .yaml files here? iow. does this load README or OWNERS ?

Reply:
good question. This function is also used to load certs and keys though. We could add a predicate maybe?

Reply:
or a pass a filename filter function, if nil then no filtering... DefaultFilter can have OWNERS and README.md

Reply:
done. Added predicates.